### ISSUE TITLE
Add number and length of comments as a feature

### ISSUE THREAD

**Issue by marco-c** [mozilla/bugbug#107]

**Comment by puneet29** [comment]
Hey, @marco-c !
Can you please clear it out whether it's a single feature for both or two seperate features. Also, how are we supposed to find the length of multiple comments for a single bug?

**Comment by marco-c** [comment]
Two features, one being the number of comments, and the other being the sum of the lengths of the comments.

**Comment by puneet29** [comment]
Thank you for replying, @marco-c 
One more question. Do I need to add these features to the feature_extractors?

#### Before adding the features:
Accuracy: f0.9324602812039998 (+/- 0.018969080793911693)
Precision: f0.9497134915934188 (+/- 0.01657777606969935)
Recall: f0.9133191208568092 (+/- 0.0320320984075)

#### After adding the features:
Accuracy: f0.931457794020608 (+/- 0.013929651276067727)
Precision: f0.9515655600681988 (+/- 0.014026640168855803)
Recall: f0.9092939444698238 (+/- 0.03103100365068092)

**Comment by marco-c** [comment]
No, for now let's just add the function in bug_features.py, but let's not use it yet in any model.
